service/src/main/kotlin/fi/espoo/evaka/application/ApplicationControllerCitizen.kt

```diff
@@ -347,7 +347,7 @@ class ApplicationControllerCitizen(
     }
 
     @DeleteMapping("/applications/{applicationId}")
-    fun deleteUnprocessedApplication(
+    fun cancelUnprocessedApplication(
```

Comment:
The action name is still `Action.Citizen.Application.DELETE`. Changing it is technically a breaking change, but I don't think there are any customizations that break if we rename it to match the name of the actual operation.